rendermodules/module/schemas/stack_schemas.py

```diff
@@ -0,0 +1,130 @@
+import warnings
+
+import argschema
+from marshmallow import ValidationError, validate, post_load, pre_load
+from .schemas import RenderParameters
+
+
+class OverridableParameterSchema(argschema.schemas.DefaultSchema):
+    @pre_load
+    def override_input(self, data):
+        return self._override_input(data)
+
+    def _override_input(self, data):
+        pass
+
+    @staticmethod
+    def fix_badkey(data, badkey, goodkey):
+        if badkey in data:
+            warnings.warn(
+                "{b} variable has been deprecated. Will try to fill in "
+                "{g} if empty.".format(b=badkey, g=goodkey),
+                DeprecationWarning)
+            data[goodkey] = (
+                data[goodkey] if (goodkey in data) else data[badkey])
+
+
+class ProcessPoolParameters(argschema.schemas.DefaultSchema):
+    pool_size = argschema.fields.Int(required=False, default=1)
+
+
+class ZValueParameters(  # argschema.schemas.DefaultSchema,
+                       OverridableParameterSchema):
+    """template schema which interprets z values on which to act
+    assumes a hierarchy such that minZ, maxZ are
+    superceded by z which is superceded by zValues.
+    """
+    minZ = argschema.fields.Int(required=False)
+    maxZ = argschema.fields.Int(required=False)
+    z = argschema.fields.Int(required=False)
+    zValues = argschema.fields.List(argschema.fields.Int, required=False)
+
+    @post_load
+    def generate_zValues(self, data):
+        return self._generate_zValues(data)
+
+    def _generate_zValues(self, data):
+        print data
```

Comment:
this print statement should be removed

Reply:
Good catch, fixed along with another one I left in.